components/dropdown/dropdown-menu.js

```diff
@@ -3,6 +3,9 @@ import { DropdownContentMixin } from './dropdown-content-mixin.js';
 import { dropdownContentStyles } from './dropdown-content-styles.js';
 import { ThemeMixin } from '../../mixins/theme/theme-mixin.js';
 
+// The duration of the delay needs to be atleast as long as the animation (300ms in this case)
+const closeTimeoutDuration = 300;
```

Comment:
I had thought this delay was for the time between when the user clicks an item and when the close animation happens, and for that case we wanted 50. Has this changed?

Reply:
Yeah, when I was testing it out further the 50ms delay didn't allow the animation to actually finish it would cut out abruptly. When I tried values that were equal to greater than the animation length the animation would have enough time to play

Using 50ms delay:

https://github.com/BrightspaceUI/core/assets/24593380/a3fe546c-0ade-4b34-a5a5-26ffae6767c3

Using 300ms delay:

https://github.com/BrightspaceUI/core/assets/24593380/3c9fa756-ee28-46b9-b99a-fc196a78f9cb